Parser/readWQMraw.m

```diff
@@ -595,18 +595,8 @@
   params{end+1} = {'DO(mmol/m^3)',    {'DOX1', ''}}; % mmol/m3 <=> umol/l
   params{end+1} = {'oxygen',          {'DOX', ''}};
   params{end+1} = {'fluorescence',    {'FLU2', ''}};
-  params{end+1} = {'F_Cal_CHL',       {'CHLF', 'Artificial chlorophyll data '...
-  'computed from bio-optical sensor raw counts measurements using factory calibration coefficient. The '...
-  'fluorometre is equipped with a 470nm peak wavelength LED to irradiate and a '...
-  'photodetector paired with an optical filter which measures everything '...
-  'that fluoresces in the region of 695nm. '...
-  'Originally expressed in ug/l, 1l = 0.001m3 was assumed.'}};
-  params{end+1} = {'U_Cal_CHL',       {'CHLU', 'Artificial chlorophyll data '...
-  'computed from bio-optical sensor raw counts measurements using user calibration coefficient. The '...
-  'fluorometre is equipped with a 470nm peak wavelength LED to irradiate and a '...
-  'photodetector paired with an optical filter which measures everything '...
-  'that fluoresces in the region of 695nm. '...
-  'Originally expressed in ug/l, 1l = 0.001m3 was assumed.'}};
+  params{end + 1} = {'F_Cal_CHL', {'CHPL', getCPHLcomment('factory', '470nm', '695nm')}};
+  params{end + 1} = {'U_Cal_CHL', {'CHPL', getCPHLcomment('user', '470nm', '685nm')}};
```

Comment:
Typo CHPL still here in the above 2 lines.

Reply:
Done